docs/examples/tensorflow/demo/nvutils/image_processing.py

```diff
@@ -137,7 +137,8 @@ def __init__(self,
         with tf.device("/gpu:0"):
             self.images, self.labels = daliop(
                 serialized_pipeline=serialized_pipe,
-                shape=[batch_size, height, width, 3],
+                shapes=[[batch_size, height, width, 3], []],
+                dtypes=[tf.float32, tf.int64],
```

Comment:
Same as above, if possible you could pass this as a tuple, as tuples are non-modifiable